### PR TITLE
Add cities to the DB

### LINKED ISSUE #57: Add a cities test data
- Add a list of all Israel cities to fill the DB with a migration file.

### PR DESCRIPTION
**Pull request by AmitAharoni**
# Description
Add the 50 biggest cities in Israel to the DB.

## Screenshots
image

### Issues close by this PR
fix: #57 

### PR Dependencies
NONE

### CODE REVIEW

**Review by Micha546 — COMMENTED** [review]
Nice 🔥

**Review by simzacks — APPROVED** [review]

File: users/migrations/0003_test_data_cities.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+from django.db import migrations, transaction
+from apartments.models import City
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('users', '0002_test_data_users'),
+        ('contacts', '0004_test_data_contacts'),
```
Comment (DanielMy777) [inline users/migrations/0003_test_data_cities.py:8]: Was this dependency auto-generated?

**Review by DanielMy777 — COMMENTED** [review]
Good job!

**Review by Omeramsc — APPROVED** [review]
LGTM
Any specific reason to have only 50? I would've tried to get a file containing of all cities in israel from one of the gov sites and use it instead.

**Comment by AmitAharoni** [comment]
@Omeramsc 
IMO there is no point in containing all the cities in the country.
Finding roommates in remote localities is quite unrealistic.
50 it's a number that maintains a balance between a respectable DB and relevance to reality.

**Pull request merged**